presto-orc/src/main/java/com/facebook/presto/orc/ColumnGroupReader.java

```diff
@@ -76,12 +78,13 @@ public ColumnGroupReader(
             int[] outputChannels,
             Map<Integer, Filter> filters,
             FilterFunction[] filterFunctions,
-            boolean reorderFilters,
-            boolean enforceMemoryBudget)
+            boolean enforceMemoryBudget,
+            Block[] constantBlocks)
     {
-        this.reorderFilters = reorderFilters;
+        this.reorderFilters = true;
         this.enforceMemoryBudget = enforceMemoryBudget;
         this.outputChannels = requireNonNull(outputChannels, "outputChannels is null");
+        this.constantBlocks = constantBlocks;
```

Comment:
requireNonNull: `this.constantBlocks = requireNonNull(constantBlocks, "constantBlocks is null");`